### DISCUSSION TITLE
How to show only H2 titles in in-page TOC

### DISCUSSION THREAD

**Discussion by tomchen** [facebook/docusaurus#3948] [Q&A] [answered]
How to show only H2 titles (instead of H2+H3) in in-page TOC (the one on the right)?

I didn't find such option. Is there a global scope option (like in docusaurus.config.js) and/or a page-scope option (like in the frontmatter)?

**Comment by slorber** [comment]
Hi,

We have the mdx loader that can support this usecase, but each plugin pass its own option to the mdx loader, and we don't have a way to configure this globally at the site level.

This is something I'd like to allow customizing, just not sure yet how this should be done

**Reply by tomchen** [reply]
OK. Actually I've read #2700 several weeks ago. I even mentioned #2700 in my #3815. Yesterday I forgot 😂

**Comment by tomchen** [comment, marked as answer]
I post this comment so I can try if I can select my own answer.

**Reply by tomchen** [reply]
Oh I can! And I even get "1 Discussion answered" under my badges on my profile page. Surprised that users can "cheat" GitHub like this. 😂😂😂

**Reply by tomchen** [reply]
Can I keep this? Sorry @slorber I should select your answer but I just want to keep mine selected for fun 🤣
